api/dna_api.go

```diff
@@ -216,13 +221,23 @@ func (api *DnaApi) Identities() []Identity {
 	return identities
 }
 
-func (api *DnaApi) Identity(address common.Address) Identity {
-	converted := convertIdentity(address, api.baseApi.getAppState().State.GetIdentity(address))
+func (api *DnaApi) Identity(addressHex string) Identity {
+	var address common.Address
+	if len(addressHex) > 0 {
+		address = common.HexToAddress(addressHex)
+	} else {
+		address = api.GetCoinbaseAddr()
+	}
+	var flipKeyWordPairs []int
+	if address == api.GetCoinbaseAddr() {
+		flipKeyWordPairs = api.ceremony.FlipKeyWordPairs()
```

Comment:
This ‘if’ is unnecessary, you can include this line in ‘else’ branch above

Reply:
Fixed